reports/ReportStructure.md

```diff
@@ -0,0 +1,18 @@
+# Structure of Reports and Explanation of Report Sections #
+### Report Name & Number ### 
+Every report has an unique number. Every report name should follow this convention:  
```

Comment:
an unique -> a unique; check with a dictionary if I am correct here and post a link to the place in the dictionary that explains the rule.